README.md

```diff
@@ -2,13 +2,57 @@
 
 This Play Store export is a job to schedule backfills of Play Store data to BigQuery via the BigQuery Data Transfer service.
 
-The purpose of this job is to continuously backfill past days over time.
-Past Play Store data has been found to still update over time
-(e.g. data from a day two weeks ago can still be updated)
+The purpose of this job is to be scheduled to run regularly in order to continuously backfill past days over time.
+Past Play Store data has been found to still update over time (e.g. data from a day two weeks ago can still be updated)
 so regular backfills of at least 30 days are required.
-The BigQuery Play Store transfer job has a non-configurable refresh
-window size of 7 days which is insufficient.
+The BigQuery Play Store transfer job has a non-configurable refresh window size of 7 days which is insufficient.
```

Comment:
This is wild to me. So basically the BQ Play store transfer job is unusable.

Reply:
From what I understand this is only an issue with the retention metrics which are forward looking up to 30 days.  I'll add that note to the readme